src/core/client/admin/App/GlobalNotification/NotificationContainer.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FunctionComponent } from "react";
+import useNotification from "./useNotification";
+
+const NotificationContainer: FunctionComponent<{}> = () => {
+  const { state } = useNotification();
+  if (!state.visible) {
+    return null;
+  }
+  return <div>{state.message}</div>;
```

Comment:
You can read this for accessibility related stuff:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role